fv3core/stencils/a2b_ord4.py

```diff
@@ -364,11 +389,20 @@ def qy_edge_south(qin: FloatField, dya: FloatFieldIJ):
 
 
 @gtscript.function
-def qy_edge_south2(qin: FloatField, dya: FloatFieldIJ, qy: FloatField):
+def qy_edge_south2(qin: FloatField, dya: FloatFieldIJ):
+    # TODO
```

Comment:
Should this also be `# TODO: when possible`, or was there a reason not to add that here?

Reply:
I think it's implied by the other TODO in the same pattern, but I can change it.